### PR TITLE
NSTF post

### PR DESCRIPTION
**Pull request by fiskanibanda**
@vukosim

### CODE REVIEW

**Review by vukosim — APPROVED** [review]
LGTM

File: _posts/2023-05-30-NSTF-South32-Awards.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "NSTF-South32 Awards 2022/2023 Finalist Announcement"
+date:   2023-05-30
+image: 
+excerpt: "We are pleased to annouce that we are a finalist for the NSTF-South32 Awards 2022/2023"
```
Comment (vukosim) [inline _posts/2023-05-30-NSTF-South32-Awards.md:6]: Maybe say, "We are pleased to announce that the Coronavirus COVID-19 (2019 – nCoV) Data Repository for South Africa project led by DSFSI is a finalist for the NSTF-South32 Awards 2022/2023"

File: _posts/2023-05-30-NSTF-South32-Awards.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "NSTF-South32 Awards 2022/2023 Finalist Announcement"
+date:   2023-05-30
+image:
```
Comment (vukosim) [inline _posts/2023-05-30-NSTF-South32-Awards.md:5]: Can you get the NSTF Logo

File: _posts/2023-05-30-NSTF-South32-Awards.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+date:   2023-05-30
+image: 
+excerpt: "We are pleased to annouce that we are a finalist for the NSTF-South32 Awards 2022/2023"
+---
+
+Data Science for Social Impact (DSFSI)  is proud to annouce that we are part of the finalists contending for the [National Science and Technology Forum (NSTF) South32 Awards 2022/2023](https://comms.evlink9.net/public/messages/view-online/LHrvhz0B2A5xjOWW/VK0aZDYDC5wSokIk/tcd2u4ZKyZ2VVp73){:target="_blank"}. These awards aim to recognise excellence and outstanding contributions to science, engineering and technology(SET) and innovation in South Africa.
+
+## Award Nomination
+
+We have nominated under the **Data for Research Award** for: Coronavirus COVID-19 (2019 – nCoV) Data Repository for South Africa (SA) – COVID19ZA Volunteers – Prof Vukosi Marivate, Principal Investigator and Leader: Data Science for Social Impact Research Group, Dept of Computer Science, University of Pretoria (UP)
```
Comment (vukosim) [inline _posts/2023-05-30-NSTF-South32-Awards.md:13]: Can you link to https://github.com/dsfsi/covid19za

See my tweets as well for other links https://twitter.com/vukosi/status/1663069119778832390?s=20

**Review by vukosim — CHANGES_REQUESTED** [review]
Please see my comments.

**Comment by fiskanibanda** [comment]
All the changes have been made @vukosim

**Review by vukosim — APPROVED** [review]
LGTM

**Pull request merged**